sqlalchemy_access/base.py

```diff
@@ -13,11 +13,18 @@
 """
 from sqlalchemy import sql, schema, types, exc, pool
 from sqlalchemy.sql import compiler, expression
+from sqlalchemy.sql.compiler import OPERATORS
+from sqlalchemy.sql.operators import ne, is_, isnot
 from sqlalchemy.engine import default, base, reflection
 from sqlalchemy import processors
 
 import pyodbc
 
+# The != operator override is required in order to make
+# the AccessCompiler class correctly format emitted queries
+OPERATORS[ne] =  " <> "
```

Comment:
won't this override the global operator in sqlalchemy and create problem if more than one dialect is used?

maybe adding a function to the compiler `def visit_ne_binary(self, binary, operator, **kw):`

Reply:
Good point. I'll have to take a look at that.

Reply:
Turns out that the "Big 5" (internal) dialects + Firebird all support both `!=` and `<>` for "not equals" so this particular change is not a show-stopper. I have implemented your suggestion and pushed it to master; it will be included in the next release. Thanks!